app/controllers/api/v2/providers_controller.rb

```diff
@@ -57,6 +57,22 @@ def publishable
         end
       end
 
+      def sync_courses_with_search_and_compare
+        authorize @provider
+        if @recruitment_cycle.year != Settings.current_recruitment_cycle.to_s
```

Comment:
maybe `RecruitmentCycle.current_recruitment_cycle` instead...

Reply:
`@recruitment_cycle.year` is a string, `Settings.current_recruitment_cycle` is apparently not.

Reply:
I meant 

```ruby
@recruitment_cycle == RecruitmentCycle.current_recruitment_cycle
```

Reply:
That wouldn't be possible because `RecruitmentCycle.current_recruitment_cycle` is an integer, however I think I've found something better.

Reply:
That isn't possible (`RecruitmentCycle.current_recruitment_cycle` is an integer) but I've found a better way.

Reply:
https://github.com/DFE-Digital/manage-courses-backend/blob/57e07c49dfe730558f52d943f3eb9cd13cdc5fb6/app/models/recruitment_cycle.rb#L31

Reply:
Scratch that, the function that's meant to do this is broken, I'll open an other PR to deal with that